### PR TITLE
Cancer participant schema changes

### PR DESCRIPTION
**Pull request by kenanM**
New in CancerParticipant 1.0.4

CancerParticipant
* Add yearOfBirth
* remove TNMStage/Grouping fields
* Add morphology field

TumourSample
* Rename tumourType -> diseaseType
* Rename tumourSubType -> diseaseSubType
* Rename phase -> tumourType
* Add 3 morphology fields
* Add 3 topography fields

### CODE REVIEW

File: schemas/IDLs/org.gel.models.participant.avro/1.0.4/VersionControl.avdl [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+/**
+This protocol defines the Participants and Pedigree
+*/
+protocol GitVersionControl {
+
+record VersionControl{
+
+    /** This is the version for the entire set of data models as referred to the Git release tag
+    */
+    string GitVersionControl="1.0.3";
```
Comment (greglever) [inline schemas/IDLs/org.gel.models.participant.avro/1.0.4/VersionControl.avdl]: can we please make this:
```
string GitVersionControl="1.0.4";
```

**Comment by kenanM** [comment]
Can we hold off on merging this. We are currently discussing whether the `morphology` field should be a string or a list of strings.

**Comment by greglever** [comment]
No worries @kenanM 
It may make sense for this to either be merged into `1.0.3-SNAPSHOT` and for `1.0.3` to be released, or for this to be released as `1.0.3` and then `1.0.3-SNAPSHOT` to become `1.0.4-SNAPSHOT`.
Did you have a preference yet @Antonior26 ?

**Comment by kenanM** [comment]
I've updated the morphologies field to be a list. @Antonior26 feel free to merge when you're ready.

**Pull request merged**